src/roster/templates/roster/partial/student_details_reading_details.html

```diff
@@ -0,0 +1,104 @@
+{% load humanize formatting %}
+<div id="readingDetailsPanel">
+    <div class="box box-secondary mt-1">
+        <div class="row gx-1 flex-items-center mb-0_5">
+            <div class="col-auto">
+                <h4 class="h2 mb-0">Reading details</h4>
+            </div>
+            <div class="col-md-auto ms-md-auto">
+                <section aria-label="Legend">
+                    <span class="readtime-legend-mark">
+                        <span class="readtime-mark-shared"></span>
+                    </span>
+                    <span class="readtime-legend-label">Assigned</span>
+                </section>
+            </div>
+        </div>
+        <div class="box-divider"></div>
+        <div class="table-scroll datatable-scroll">
+            <table class="table datatable">
+                <thead>
+                    <!-- Available sort indicators for table header anchors:
+                        - `.sort-up`
+                        - `.sort-down`
+                        - `.sort-active`
+                    -->
+                    <tr>
+                        <th><a href="#" data-sort="title" data-pagenum="{{ data.page_obj.number }}" data-username="{{ data.username }}" data-days="{{ data.days }}" class="reading-details-link{% if data.sort == 'title' %} sort-active{% endif %}">Title</a>/author</th>
+                        <th><a href="#" data-sort="time" data-pagenum="{{ data.page_obj.number }}" data-username="{{ data.username }}" data-days="{{ data.days }}" class="reading-details-link{% if data.sort == 'time' %} sort-active{% endif %}">Active time</a></th>
+                        <th><a href="#" data-sort="lastview" data-pagenum="{{ data.page_obj.number }}" data-username="{{ data.username }}" data-days="{{ data.days }}" class="reading-details-link{% if data.sort == 'lastview' %} sort-active{% endif %}">Last read</a></th>
+                        <th class="d-sm-down-none">Words looked up</th>
+                        <th class="d-sm-down-none">Learning</th>
+                        <th class="d-md-down-none">Custom</th>
+                        <th class="d-md-down-none">Reading level</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for book in data.page_obj %}
+                    <tr>
+                        <th scope="row">
+                            <a href="{% url 'reader_default' book_id=book.book_id %}">{{ book.title }}</a>
```

Comment:
We will need to handle Bookshare books that the teacher is not able to view. Can borrow this code & design from dashboard.